firmware/src/PeO.h

```diff
@@ -0,0 +1,20 @@
+#ifndef PeO_H_
+#define PeO_H_
+
+#include"conf.h"
+#include "pwm.h"
+#define PWM_D_STEP 					        1
+
+#define MINIMUM_PANEL_VOLTAGE_MAX_POWER  9.0f
+extern volatile float max_power;
+extern volatile float max_power_duty_cycle;
+
+typedef enum state_PeO{
+    increase,
+    decrement,
+} state_PeO_t;
+state_PeO_t state_PeO;
```

Comment:
avoid using lower case constants, and also enums or constants that are only used on the file is a good pratice start the name with <filename>_<var name> for example PEO_INCREMENT